docs/guides/getting_started.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: "mcs"
+page_title: "Getting Started with the MCS Provider"
+description: |-
+  Getting Started with the MCS Provider
+---
+
+# Create basic config for MCS Provider resources
+
+This example shows how to create a simple terraform configuration for creation of MCS resources.
+
+First, create a Terraform config file named `main.tf`. Inside, you'll want to include the configuration of
+[MCS Provider](/terraform/docs/providers/mcs/index.html),
+[Openstack Provider](https://www.terraform.io/docs/providers/openstack/index.html),
+[openstack_networking_network_v2](https://www.terraform.io/docs/providers/openstack/d/networking_network_v2.html),
+[openstack_networking_subnet_v2](https://www.terraform.io/docs/providers/openstack/r/networking_subnet_v2.html),
+[openstack_networking_network_v2](https://www.terraform.io/docs/providers/openstack/d/networking_network_v2.html),
+[openstack_networking_router_v2](https://www.terraform.io/docs/providers/openstack/r/networking_router_v2.html),
+[openstack_networking_router_interface_v2](https://www.terraform.io/docs/providers/openstack/r/networking_router_interface_v2.html),
+[openstack_compute_keypair_v2](https://www.terraform.io/docs/providers/openstack/d/compute_keypair_v2.html)
+and [openstack_compute_flavor_v2](https://www.terraform.io/docs/providers/openstack/d/compute_flavor_v2.html).
+
+Use MCS provider:
+
+```hcl
+provider "mcs" {
+    username   = "some_user"
+    password   = "s3cr3t"
+    project_id = "some_project_id"
+    auth_url   = "https://infra.mail.ru/identity/v3/"
```

Comment:
let's remove `auth_url` from examples, the provided value is a default const. `auth_url` is mentioned as optional param here https://registry.terraform.io/providers/MailRuCloudSolutions/mcs/latest/docs and I guess that's enough.

Reply:
Fixed